integration-tester/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,77 @@
+package kubernetes
```

Comment:
I added a Kubernetes module to `workflow-manager` in #292 that shares a little bit of functionality with this file, namely the bits that get a client. It'd be nice to share modules between different go targets.

Reply:
I'll wait for that to get merged in and pull that module in here

Reply:
Yep it's fine! Just when building in docker the entire context of the git repo needs to be sent to the build agent, rather than this project alone.